components/app/header/Banner.vue

```diff
@@ -18,13 +18,13 @@ function removeBanner() {
 </script>
 
 <template>
-  <div v-show="showBanner && bannerConfig?.show"
+  <div v-show="showBanner && bannerConfig && bannerConfig.show"
     class="flex items-center gap-x-6 bg-primary-600 px-6 py-2.5 sm:px-3.5 sm:before:flex-1">
     <p class="text-sm leading-6 text-white">
       <a :href="href" target="_blank" class="inline-flex items-center gap-1">
-        <strong v-if="bannerConfig?.brandText" class="font-semibold hidden sm:block">{{ bannerConfig.brandText
-          }}</strong>
-        <AppHeaderDot class="hidden sm:block" v-if="bannerConfig?.brandText" />
```

Comment:
The dot was removed from the header ?

Reply:
Yeah correct, there is not AppHeaderDot Component in the component tree, probably the component that this refer to can be ContentDots.vue

Reply:
Ok, I will bring that component back.

It's separating the two texts in the banner:
<img width="1134" alt="Screenshot 2024-10-01 at 09 44 37" src="https://github.com/user-attachments/assets/74cc6f0b-0e03-4b5b-b41f-538fdf00cb2a">